### PR TITLE
replication to create time date directory

### PR DESCRIPTION
**Pull request by kmlebedev**
replace https://github.com/chrislusf/seaweedfs/pull/1761

### CODE REVIEW

File: weed/command/scaffold.go [hunk 1]
```diff
@@ -353,9 +353,14 @@ directory = "/buckets"
 
 [sink.local]
```
Comment (chrislusf) [inline weed/command/scaffold.go:354]: how about use these two sink names, "local_incremental" and "local"?

Comment (kmlebedev) [reply weed/command/scaffold.go:354]: ok

**Comment by kmlebedev** [comment]
done

**Comment by chrislusf** [comment]
I think there are some errors. Let me merge and check.

**Comment by chrislusf** [comment]
I updated the related code. Please help to check whether it makes sense.

**Comment by kmlebedev** [comment]
> I updated the related code. Please help to check whether it makes sense.

https://github.com/chrislusf/seaweedfs/commit/da08402ba253fe5391f0152905f51f82c51fd527#commitcomment-46462627

**Pull request merged**